bin/whitelist

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+#
+# Copyright 2017 Intel Corporation
```

Comment:
Should the copyright be updated since this is a new file?

Reply:
It's a copy of the whitelist file from sawtooth-core so I believe the original copyright is still correct.